modules/dev-server/main.tf

```diff
@@ -173,7 +243,51 @@ exec consul agent -server \
   -hcl 'telemetry { disable_compat_1.9 = true }' \
   -hcl 'connect { enabled = true }' \
   -hcl 'enable_central_service_config = true' \
+%{if var.tls~}
+  -hcl='ca_file = "/consul/consul-agent-ca.pem"' \
+  -hcl='cert_file = "/consul/dc1-server-consul-0.pem"' \
+  -hcl='key_file = "/consul/dc1-server-consul-0-key.pem"' \
+  -hcl='auto_encrypt = {allow_tls = true}' \
+  -hcl='ports { https = 8501 }' \
```

Comment:
should we set `http = -1`?

Reply:
I don't see a reason to do that since it's a dev server. It makes our changes much easier in that we don't need to change the load balancer. Since we're only using it for testing, we thought it's not worth the complexity.

Reply:
Oh yeah totally agreed!